airflow/providers/openlineage/provider.yaml

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+package-name: apache-airflow-providers-openlineage
+name: OpenLineage Airflow
+description: |
+  `OpenLineage <https://openlineage.io/>`__
+
+versions:
+  - 1.0.0
+
+dependencies:
+  - apache-airflow>=2.5.1
```

Comment:
This needs to be set as the most recent published Airflow version. Ideally, we'd want to have it set to the first version released with any core changes needed to make OpenLineage provider working (not sure if it'll be 2.5.2 or 2.6.0)

Reply:
Not something we should focus on now. I will not release the provider before the AIP is complete so we can update this one later.
OL support is considered a feature thus core changes related to it will be in 2.6 or 2.7